scalacheck/src/main/scala/pathy/scalacheck/RandomSeg.scala

```diff
@@ -28,9 +28,10 @@ private[scalacheck] object RandomSeg {
   implicit val randomSegArbitrary: Arbitrary[RandomSeg] =
     Arbitrary {
       Gen.nonEmptyListOf(Gen.frequency(
-        100 -> Arbitrary.arbitrary[Char],
+        100 -> Gen.alphaNumChar,
```

Comment:
All I know is that when converting web into the Free architecture, we also transformed most of the tests into properties and that exposed many bugs that were latent in the previous implementation. So much so, that I had to go with `PathOf[AlphaCharacters]` in many places just to move on and complete the translation in a timely manner. I don't know if we will ever find the time to change the properties back to `PathOf[RandomSeg]` or whether that makes sense, but my concern is that we might not do as good of a job exposing problems as many of the tests would only break every so often, so presumably the corner cases were hard to find/generate. On the other hand, I do believe they were probably mostly caused by the `escapedChars` so I can't say for sure that keeping many truly arbitrary characters would help find these corner cases.

I don't have any strong argument against this change and the important thing for me is that it's relatively easy to customize so I don't want to spend an exaggerated amount of time discussing what is the most appropriate default behavior. So as far as I am concerned, I'm okay with this change as is.